packages/foundry/foundry.toml

```diff
@@ -0,0 +1,21 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+fs_permissions = [{ access = "read-write", path = "./"}]
+
+[rpc_endpoints]
+default_network = "http://127.0.0.1:8545"
+localhost = "http://127.0.0.1:8545"
+
+mainnet = "https://eth-mainnet.alchemyapi.io/v2/${providerApiKey}"
+sepolia = "https://eth-sepolia.g.alchemy.com/v2/${providerApiKey}"
+goerli = "https://eth-goerli.alchemyapi.io/v2/${providerApiKey}"
+arbitrum = "https://arb-mainnet.g.alchemy.com/v2/${providerApiKey}"
+arbitrumGoerli = "https://arb-goerli.g.alchemy.com/v2/${providerApiKey}"
+optimism = "https://opt-mainnet.g.alchemy.com/v2/${providerApiKey}"
+optimismGoerli = "https://opt-goerli.g.alchemy.com/v2/${providerApiKey}"
+polygon = "https://polygon-mainnet.g.alchemy.com/v2/${providerApiKey}"
+polygonMumbai = "https://polygon-mumbai.g.alchemy.com/v2/${providerApiKey}"
```

Comment:
Would the ${providerApiKey} work here? How do I set up the .env var?

Reply:
it wont work here, but a .toml file is not usually integrated with .env files. for now keeping it this way is the best. it wont break, you will need to replace your api key in this file.